File: Euler-Particles/src/color/HSB/CoolColorGenerator.java
package color.HSB;

import processing.core.PApplet;

public class CoolColorGenerator extends HSBColorGenerator {

    public CoolColorGenerator(PApplet p) {
        super(p);
    }

    public int mapColor(float value, float minValue, float maxValue, int saturation, int brightness) {
        p.colorMode(PApplet.HSB, 360);
        int hue = (int)PApplet.map(value, minValue, maxValue, 90, 285);
        return p.color(hue, saturation, brightness);
    }

}

File: docs/Acceleration/color/hsb/cool_color_generator.js
class CoolColorGenerator extends HSBColorGenerator {

    mapColor(value, minValue, maxValue, saturation, brightness) {
        colorMode(HSB, 360);
        let hue = int(map(value, minValue, maxValue, 90, 285));
        return color(hue, saturation, brightness);
    }
}
